bats/operations.bash

```diff
@@ -0,0 +1,108 @@
+create_deposit_tx_template_variables() {
```

Comment:
I think this is not `operations` but `tx-templates`

Actually how about you just hard code this in the `tx-template-create.gql` file instead of putting all then inputs via bash / json.

You just need to inject the id I guess.